src/AspNet.Security.OAuth.Kook/KookAuthenticationOptions.cs

```diff
@@ -0,0 +1,35 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+
+namespace AspNet.Security.OAuth.Kook;
+
+/// <summary>
+/// Defines a set of options used by <see cref="KookAuthenticationHandler"/>.
+/// </summary>
+public class KookAuthenticationOptions : OAuthOptions
+{
+    public KookAuthenticationOptions()
+    {
+        ClaimsIssuer = KookAuthenticationDefaults.Issuer;
+        CallbackPath = KookAuthenticationDefaults.CallbackPath;
+        AuthorizationEndpoint = KookAuthenticationDefaults.AuthorizationEndpoint;
+        TokenEndpoint = KookAuthenticationDefaults.TokenEndpoint;
+        UserInformationEndpoint = KookAuthenticationDefaults.UserInformationEndpoint;
+
+        ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "id");
+        ClaimActions.MapJsonKey(ClaimTypes.Name, "username");
+        ClaimActions.MapJsonKey(ClaimTypes.MobilePhone, "mobile");
+        ClaimActions.MapJsonKey(KookAuthenticationConstants.Claims.IdentifyNumber, "identify_num");
+        ClaimActions.MapJsonKey(KookAuthenticationConstants.Claims.OperatingSystem, "os");
+        ClaimActions.MapJsonKey(KookAuthenticationConstants.Claims.AvatarUrl, "avatar");
+        ClaimActions.MapJsonKey(KookAuthenticationConstants.Claims.BannerUrl, "banner");
```

Comment:
`os` and `banner` don't appear to be present in the documentation.

![image](https://user-images.githubusercontent.com/1439341/232286738-cb0dd406-0a6d-44b2-85f3-566fac067697.png)

Reply:
The documentation is maintained by officials and developers together. The content may be not up to date. I will commit some pull requests to fix the unrenewed content soon.

By testing the actual behaviors, the API returns the content as below:

![image](https://user-images.githubusercontent.com/21241496/232319762-4377904c-6c4b-4774-a8df-a5083c3673bf.png)

By testing via the Mvc.Client project, I could get the authorization as below:

![image](https://user-images.githubusercontent.com/21241496/232319855-e9ee31a1-cefa-43ee-9ac2-bdfda907f88f.png)

Reply:
Documentation fixture pull request has been made: kaiheila/api-docs#184